.github/workflows/diff-change-to-package.yaml

```diff
@@ -32,15 +32,15 @@ jobs:
 
       - name: Generate diff
         run: |
-          # Using `origin/$GITHUB_BASE_REF` to avoid actually checking out the branch
-          # as all we need is to let Git diff the two references
-          bin/dist-diff.sh origin/$GITHUB_BASE_REF $GITHUB_WORKSPACE
+          git config user.name github-actions
```

Comment:
Ah this is a long story

We know we want to use `git diff` for [`js-beautify` formatting](https://github.com/beautify-web/js-beautify) but the npm package isn't committed

### Running `git diff` on uncommitted files

Since we need to build both the "before" (base) and "after" (head) npm packages:

**Idea 1:** Build into two directories and run [`git diff --no-index`](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-emgitdiffemltoptionsgt--no-index--ltpathgtltpathgt)
**Idea 2:** Build + commit twice, then run `git diff HEAD^ HEAD` on the last 2 commits

Sadly we can't use [`":(exclude)**/*.min.*"` pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec) with `git diff --no-index`

But to commit each build we need `user.name` and `user.email` even though they never go anywhere